HWFlask-2.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, render_template, request, redirect, make_response
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def index():
+    return render_template("index.html")
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    name = request.form.get("name")
+    email = request.form.get("email")
+
+    resp = make_response(redirect("/hello"))
```

Comment:
лучше использовать функцию `url_for`
```suggestion
    resp = make_response(redirect(url_for("hello")))
```